### ISSUE TITLE
Latest version not starting

### ISSUE THREAD

**Issue by crustulumtheoneandonly** [unpoller/unpoller#443]
Docker logs unifipoller:

`2022/11/23 15:26:15 [ERROR] finding file: stat /etc/unifi-poller/up.conf: no such file or directory`

docker-compose.yml :

```
version: '3.8'
services:

  unifipoller:
    container_name: unifi-poller
    restart: always
    image: ghcr.io/unpoller/unpoller:latest
    environment:
      - UP_INFLUXDB_DB=unifipoller
      - UP_INFLUXDB_USER=USER
      - UP_INFLUXDB_PASS=PASS
      - UP_INFLUXDB_URL=http://DB
      - UP_UNIFI_DEFAULT_USER=USER
      - UP_UNIFI_DEFAULT_PASS=PW
      - UP_UNIFI_DEFAULT_URL=https://IP
      - UP_POLLER_DEBUG=false
      - UP_UNIFI_DEFAULT_SAVE_DPI=true

```

**Comment by platinummonkey** [comment]
hm this looks to come from `unpoller/poller` default config no longer can continue without at least an empty file existing 🤔

**Comment by cdalvaro** [comment]
Is this fixed in the latest release (9.3.0)??

**Comment by platinummonkey** [comment]
not yet

**Comment by qbatten** [comment]
Running into this with v2.4.0 as well.

**Comment by platinummonkey** [comment]
Yes this won’t be fixed yet in that release, I’ll be looking at this next though

**Comment by tuxthepenguin84** [comment]
The latest docker image v2.4.0 starts but does not start collecting data. Reverting to v2.3.1 works fine.

**Comment by platinummonkey** [comment]
was able to test this myself, should be fixed now in 2.4.1
